examples/riot-kitchen-sink/.storybook/addons.js

```diff
@@ -0,0 +1,8 @@
+import '@storybook/addon-storysource/register';
```

Comment:
Looks like not all of these addons are marked as supported in ADDONS_SUPPORT.md

Reply:
Yes I should try again with links